internal/pkg/api/storageapi/error.go

```diff
@@ -8,11 +8,7 @@ import (
 
 type ErrorWithResponse interface {
 	SetResponse(response *resty.Response)
-	HttpStatus() int
-	IsBadRequest() bool
-	IsUnauthorized() bool
-	IsForbidden() bool
-	IsNotFound() bool
+	StatusCode() int
 }
```

Comment:
Tento interface som spravil jednoduchsim, pouzivala sa iba metoda `IsUnauthorized` aj to iba 1x.